docs/resources/group.md

```diff
@@ -0,0 +1,47 @@
+---
+page_title: "Resource hcp_group - terraform-provider-hcp"
+subcategory: "Cloud Platform"
+description: |-
+  The group resource manages a HCP Group.
+  The user or service account that is running Terraform when creating an hcp_group resource must have roles/admin on the parent resource; either the project or organization.
+---
+
+# hcp_group (Resource)
+
+The group resource manages a HCP Group.
+
+The user or service account that is running Terraform when creating an `hcp_group` resource must have `roles/admin` on the parent resource; either the project or organization.
+
+## Example Usage
+
+```terraform
+resource "hcp_project" "example" {
+  display_name = "example-group"
+  description  = "My new group!"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `display_name` (String) The group's display_name.
+
+### Optional
+
+- `description` (String) The group's description
+
+### Read-Only
+
+- `resource_id` (String) The group's unique identifier
+- `resource_name` (String) The group's resource name in the format `iam/organization/<organization_id>/group/<name>` or `<name>`
```

Comment:
@squaresurf I didn't include the `or 'name'` bit in the resources docs here since the `resource_name` is computed rather than an input here and always would be the full version of the `resource_name`.

Reply:
Nice catch. Thanks @squaresurf!